libr/core/cmd_anal.c

```diff
@@ -6552,21 +6559,63 @@ static void cmd_anal_esil(RCore *core, const char *input) {
 			switch (input[2]) {
 			case ' ': // "aeli" with arguments
 				if (!r_anal_esil_load_interrupts_from_lib (esil, input + 3)) {
-					eprintf ("Failed to load interrupts from '%s'.", input + 3);
+					eprintf ("Failed to load interrupts from '%s'.\n", input + 3);
 				}
 				break;
 			case 0: // "aeli" with no args
 				if (esil && esil->interrupts) {
+					if (esil->intr0) {
+						_aeli_print (0ULL, esil->intr0);
+					}
 					dict_foreach (esil->interrupts, _aeli_iter, NULL);
 				}
 				break;
 			case 'r': // "aelir"
 				if (esil && esil->interrupts) {
-					RAnalEsilInterrupt* interrupt = dict_getu (esil->interrupts, r_num_math (core->num, input + 3));
-					r_anal_esil_interrupt_free (esil, interrupt);
+					const ut32 intr_num = r_num_math (core->num, input + 3);
+					if (intr_num) {
+						RAnalEsilInterrupt* interrupt = dict_getu (esil->interrupts, intr_num);
+						r_anal_esil_interrupt_free (esil, interrupt);
+					} else {
+						r_anal_esil_interrupt_free (esil, esil->intr0);
+					}
+				}
+				break;
+			}
+			break;
+		case 's': // aels interrupts
+			switch (input[2]) {
+			case ' ': // "aels" with arguments
+				if (!r_anal_esil_load_syscalls_from_lib (esil, input + 3)) {
+					eprintf ("Failed to load syscalls from '%s'.\n", input + 3);
+				}
+				break;
+			case 0: // "aels" with no args
+				if (esil && esil->syscalls) {
+					if (esil->sysc0) {
+						_aeli_print (0ULL, esil->sysc0);
+					}
+					dict_foreach (esil->syscalls, _aeli_iter, NULL);
+				}
+				break;
+			case 'r': // "aelsr"
+				if (esil && esil->syscalls) {
+					const ut32 sysc_num = r_num_math (core->num, input + 3);
+					if (sysc_num) {
+						RAnalEsilSyscall *syscall = dict_getu (esil->syscalls, sysc_num);
```

Comment:
Use `sc` instead of `syscall` which is defined in unistd.h and may result on compilation issues or reader confussion